USAGE_android.md

```diff
@@ -260,6 +514,215 @@ originally named `gfxrecon_capture.gfxr` and was created at 2:35 PM
+
+The current version of the Android capture layer has the following limitations:
+
+ * The capture layer must be deployed and enabled manually.
+   Future work may provide a tool to automate the process of launching a
+   debuggable app, injecting the trace layer with the Java Debug Wire Protocol
+   (JDWP), and enabling the layer.
+ * Writing files to external storage requires the external storage permission.
+   If the app to be captured does not already have this permission, it can be
+   granted manually through the Applications panel from device Settings.
+ * Adding a timestamp to the filename does not guarantee a unique filename.
+   Some applications will query for Vulkan support by creating an instance,
+   retrieving instance properties, and then destroying the instance.
+   After the initial support query, the application will create a second
+   instance to do its real work.
+   Each of these Vulkan sessions will generate a capture file.
+   If the time elapsed between these two sessions is less than a second, both
+   sessions will use the same file name, with the second session overwriting the
+   file produced by the first session.  Future improvements to the capture layer
+   will include a method of checking for an existing file and appending a unique
+   sequence number to prevent overwriting an existing file.
+
+### Known Capture Issues
+
+#### Conflicts With Crash Detection Libraries
+
+As described by the above
+[Understanding GfxReconstruct Layer Memory Capture](#understanding-gfxreconstruct-layer-memory-capture)
+section, the capture layer uses a signal handler to detect modifications to
+mapped memory.
+Only one signal handler can be registered at a time, which can lead to a
+potential conflict with crash detection libraries that will also register a
+signal handler.
+
+Conflict between the capture layer and crash detection libraries depends on the
+order with which each component registers its signal handler.
+The capture layer will not register its signal handler until the first call to
+`vkMapMemory`.
+As long as the application initializes the crash detection library before
+calling `vkMapMemory`, there should be no conflict.
+
+The conflict occurs when the application initializes its Vulkan component and
+its crash detection library concurrently.
+Applications have been observed to initialize Vulkan and begin uploading
+resources with one or more threads, while at the same time initializing a crash
+detection library from another thread.
+For this scenario, the crash detection library sets its signal handler after the
+first call to `vkMapMemory`, while a resource upload thread is actively writing
+to the mapped memory.
+After the crash detection library sets its signal handler, it immediately
+receives a SIGSEGV event generated by the concurrent write to mapped memory,
+which it detects as a crash and terminates the application.
+
+This may be a problem that is difficult to solve reliably.
```

Comment:
I think you could probably omit this - it's kind of editorial and academic and doesn't really help with usage of the tool.